### PR TITLE
Add coreos NCP profile

### PR DESCRIPTION
**Pull request by redhatrises**
#### Description:

- Adds new CoreOS draft ncp profile. Everything in the profile is subject to change, and the profile is
  purposefully not buildable for now.

### CODE REVIEW

File: coreos/profiles/ncp.profile [hunk 1]
```diff
@@ -0,0 +1,490 @@
+documentation_complete: true
+
+title: 'NIST National Checklist Program Security Guide'
```
Comment (shawndwells) [inline coreos/profiles/ncp.profile]:
```suggestion
title: 'NIST National Checklist for Red Hat Enterprise Linux CoreOS'
```

Comment (shawndwells) [reply coreos/profiles/ncp.profile]: Ideally, a version number or range (e.g. 8.x) should be included.

Comment (redhatrises) [reply coreos/profiles/ncp.profile]: Versioning will probably change so much that this would be an exercise in futility.

Comment (shawndwells) [reply coreos/profiles/ncp.profile]: Yeah, you're right, and I see they've been removed for other profiles. Looks good to me!

**Pull request merged**